k4FWCore/scripts/k4run

```diff
@@ -171,4 +171,7 @@ if __name__ == "__main__":
     if not opts.dry_run:
         # Do the real processing
         retcode = c.run(opts.gdb, opts.ncpus)
+        # User requested stop returns non-zero exit code see: https://github.com/key4hep/k4FWCore/issues/125
+        if ApplicationMgr().EvtMax == -1 and retcode == 0x04:
+           retcode = 0x00
```

Comment:
Ugh can we use at least `retcode = 0` instead of `0x00`, the other one I care less because it's coming from Gaudi but it looks so ugly

Reply:
0x00 is also from Gaudi ;) but sure, I can change them. :)